src/constants.js

```diff
@@ -6,45 +6,137 @@
 export const LANDING_PAGE_TITLE = 'COLLECTIVA';
 
 // Name for all the lines
+// TODO: maybe deprecate this in favor of the more stacked LINE_INFO
 export const LINE_NAMES = [
```

Comment:
I think I deprecated this already in the code so if you can delete it that would be great